external_search.py

```diff
@@ -2663,57 +2664,12 @@ def __contains__(self, k):
 
     def to_dict(self):
         return {
-            "title": self.title,
-            "author": self.author,
+            "title": self.sort_title,
+            "author": self.sort_author,
             "id": self.meta["id"],
             "meta": self.meta,
         }
 
-class SearchIndexMonitor(WorkSweepMonitor):
```

Comment:
Since bin/repair/search_index now deletes all the search coverage records and then recreates coverage, there's no need for `SearchIndexMonitor`, which was a less reliable way of doing the same thing.